cohorts/load.py

```diff
@@ -80,6 +86,18 @@ def __init__(self,
         self.nonsynonymous_effect_cache_name = "cached-nonsynonymous-effects"
         self.neoantigen_cache_name = "cached-neoantigens"
 
+    def verify_survival(self):
```

Comment:
Sorry, I think I just read the assert wrong. Maybe the min is confusing me - it should be max?